pkg/oc/bootstrap/docker/openshift/servicecatalog.go

```diff
@@ -39,18 +38,7 @@ func (h *Helper) InstallServiceCatalog(f *clientcmd.Factory, configDir, publicMa
 		return err
 	}
 
-	for _, role := range GetServiceCatalogRBACDelta() {
```

Comment:
@jboyd01 leave this as-is (there should be no changes to this file other than the `GetServiceCatalogRBACDelta` to `GetServiceCatalogClusterRoles` name change).

Reply:
Thanks, finally got it.  Tests look good.